product_docs/docs/pem/8/installing/dependencies.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: "Dependencies of the PEM Server and Agent on Linux"
+navTitle: "Linux dependencies"
+redirects:
+- /pem/latest/installing_pem_server/pem_server_inst_linux/dependencies/
+---
+
+The PEM Server and Agent packages for Linux have dependencies on various system libraries. 
+These dependencies are detailed below for reference. 
+
+!!! Note
+A PEM Agent is always installed alongside PEM Server, so all dependencies must be present on hosts where PEM Server (either the database or the web application) is installed.
+!!!
+
+Typically, PEM is built against the latest version of each dependency available from the vendor repository for a given platform and architecture.
+In some cases, PEM requires a newer version of a library than is available in the vendor repository.
+In these cases a newer version of the package, prefixed with `edb-` is sourced from EDB's repositories.
+
+!!! Note
+This information is provided for reference. Packages from vendor repositories are not supported or patched by EDB. 
+Refer to your operating system documentation or support provider for details of these packages.  
+
+Because these dependencies are updated frequently, the tables below are valid only for the latest patch release of PEM.
```

Comment:
I don't think this page should exist in PEM 8. In any case, PEM 8 is EOL so we will be looking to remove the docs very soon. We could do that now if it helps with this change?

Reply:
deleted